trpcultivate_germplasm/src/Plugin/TripalImporter/GermplasmAccessionImporter.php

```diff
@@ -0,0 +1,844 @@
+
+    $columns = [
+      'Germplasm Name' => 'Name of this germplasm accession (e.g. CDC Redberry)',
+      'External Database' => 'The institution who assigned the accession. (e.g. KnowPulse Germplasm)',
+      'Accession Number' => 'A unique identifier for the accession (e.g. KP:GERM58)',
+      'Germplasm Species' => 'The species of the accession (e.g. culinaris)',
+      'Germplasm Subtaxa' => 'The rank below species is specified first, followed by the name (e.g. var. medullare)',
+      'Institute Code' => 'The code for the Institute that bred the material (e.g. CUAC)',
+      'Institute Name' => 'The name of the Institute that bred the material (e.g. "Crop Development Center, University of Saskatchewan")',
+      'Country of Origin Code' => '3-letter ISO 3166-1 code of the country in which the sample was originally sourced (e.g. 124)',
+      'Biological Status of Accession' => 'The 3 digit code representing the biological status of the accession (e.g. 410)',
+      'Breeding Method' => 'The unique identifier for the breeding method used to create this germplasm (e.g. "Recurrent selection")',
+      'Pedigree' => 'The cross name and optional selection history (e.g. 1049F^3/819-5R)',
+      'Synonyms' => 'Any synonyms of the accession, separated by a comma. (e.g. Redberry)',
+    ];
+
+    $required_col = ['Germplasm Name', 'External Database', 'Accession Number', 'Germplasm Species'];
+    // @TODO: Make this red, but Drupal makes it difficult :)
+    $required_markup = '*';
+
+    $output .= '<ol>';
+    foreach ($columns as $title => $definition) {
+      if (in_array($title, $required_col)) {
+        $output .= '<li><b>' . $title . $required_markup . '</b>: ' . $definition . '</li>';
+      }
+      else {
+        $output .= '<li><b>' . $title . '</b>: ' . $definition . '</li>';
+      }
+    }
+    $output .= '</ol>';
+
+    return $output;
+  }
+
+  /**
+   * Set a cvterm with its cvterm_id
+   *
+   * @param string $key
+   *   A key used in the config settings.yml
+   * @param int $cvterm_id
+   * @return TRUE
+   */
+  public function setCVterm($key, $cvterm_id) {
+    $this->cvterms[$key] = $cvterm_id;
+    return TRUE;
+  }
+
+  /**
+   * Get a cvterm ID, given a key that maps to the config settings.yml
+   *
+   * @param string $key
+   *   The cvterm name
+   * @return int
+   *   The cvterm ID
+   */
+  public function getCVterm($key) {
+    return $this->cvterms[$key];
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function form($form, &$form_state) {
+    $form = parent::form($form, $form_state);
+
+    // Select the entire genus field and make sure it is sorted and distinct
+    $genus_query = $this->connection->select('1:organism', 'o')
+      ->fields('o',['genus'])
+      ->orderBy('genus')
+      ->distinct();
+    $genus = $genus_query->execute()->fetchAllKeyed(0,0);
+
+    $form['instructions'] = [
```

Comment:
This feels a bit out of place 🤔 
![Screenshot 2023-12-12 at 3 22 52 PM](https://github.com/TripalCultivate/TripalCultivate-Germplasm/assets/1566301/9a25c59d-41a8-476c-b623-64519db6ca5c)

What about instead:

> Import germplasm accessions into Chado with metadata meeting BrAPI standards using this form and the associated data file. **Please confirm the file format and column order before upload as this will insert records into your Chado database.**